docs/user_guide/architecture.rst

```diff
@@ -457,6 +433,44 @@ AMD platforms
    connected to the FMC, the custom VADJ will have the value supported by
    both the carrier and the device(s)
 
+Discontinued AMD platforms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
```suggestion

Projects with these carriers can still be found on our repository, as well as their :git-hdl:`board files <projects/common>`.
```
Could you put a similar text below the "AMD platforms" section, stating "The board files of these carriers can be found  :git-hdl:\`here <projects/common>\`."